apps/web/playwright/webhook.e2e.ts

```diff
@@ -543,6 +544,131 @@ test.describe("BOOKING_RESCHEDULED", async () => {
+    ]);
+
+    const scheduledTriggers = await prisma.webhookScheduledTriggers.findMany({
+      where: {
+        webhook: {
+          userId: user.id,
+        },
+      },
+      select: {
+        payload: true,
+        webhook: {
+          select: {
+            userId: true,
+            id: true,
+            subscriberUrl: true,
+          },
+        },
+        startAfter: true,
+      },
+    });
+
+    const existingUserBookings = await prisma.booking.findMany({
+      where: {
+        userId: user.id,
+        startTime: {
+          gt: new Date(),
+        },
+      },
+    });
+
+    const meetingStartedTriggers = scheduledTriggers.filter((trigger) =>
+      trigger.payload.includes("MEETING_STARTED")
+    );
+    const meetingEndedTriggers = scheduledTriggers.filter((trigger) =>
+      trigger.payload.includes("MEETING_ENDED")
+    );
+
+    expect(meetingStartedTriggers.length).toBe(existingUserBookings.length);
+    expect(meetingEndedTriggers.length).toBe(existingUserBookings.length);
+
+    expect(meetingStartedTriggers.map((trigger) => trigger.startAfter)).toEqual(
+      expect.arrayContaining(existingUserBookings.map((booking) => booking.startTime))
+    );
+    expect(meetingEndedTriggers.map((trigger) => trigger.startAfter)).toEqual(
+      expect.arrayContaining(existingUserBookings.map((booking) => booking.endTime))
+    );
+
+    page.reload();
+
+    // edit webhook and remove trigger meeting ended trigger
+    await page.click('[data-testid="webhook-edit-button"]');
+    await page.getByRole("button", { name: "Remove Meeting Ended" }).click();
+
+    await Promise.all([
+      page.click("[type=submit]"),
+      page.waitForURL((url) => url.pathname.endsWith("/settings/developer/webhooks")),
+    ]);
+
+    const scheduledTriggersAfterRemovingTrigger = await prisma.webhookScheduledTriggers.findMany({
+      where: {
+        webhook: {
+          userId: user.id,
+        },
+      },
+    });
+
+    const newMeetingStartedTriggers = scheduledTriggersAfterRemovingTrigger.filter((trigger) =>
+      trigger.payload.includes("MEETING_STARTED")
+    );
+    const newMeetingEndedTriggers = scheduledTriggersAfterRemovingTrigger.filter((trigger) =>
+      trigger.payload.includes("MEETING_ENDED")
+    );
+
+    expect(newMeetingStartedTriggers.length).toBe(existingUserBookings.length);
+    expect(newMeetingEndedTriggers.length).toBe(0);
+
+    // disable webhook
+    await page.click('[data-testid="webhook-switch"]');
+
+    await page.waitForLoadState("networkidle");
+
+    const scheduledTriggersAfterDisabling = await prisma.webhookScheduledTriggers.findMany({
+      where: {
+        webhook: {
+          userId: user.id,
+        },
+      },
+      select: {
+        payload: true,
+        webhook: {
+          select: {
+            userId: true,
+          },
+        },
+        startAfter: true,
+      },
+    });
+
+    expect(scheduledTriggersAfterDisabling.length).toBe(0);
```

Comment:
We're receiving a 2 in here:

<img width="764" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/5e239d61-1133-40aa-a06f-e59488539b4f">